docs/topics/proc_installing-launcher-tool-addon.adoc

```diff
@@ -15,6 +15,14 @@ link:https://docs.openshift.org/latest/minishift/using/addons.html[Add-ons] enab
 
 .Procedure
 
+. Ensure you are running a minishift instance which has at least 4 GB memory to run launcher
+
+[source,bash,options="nowrap",subs="attributes+"]
+----
+$ minishift config set memory 8192
+$ minishift start
+---
```

Comment:
can you make this `----` (four dashes) so it matches the start of the block (this actually cause the build to break)